examples/standalone_mode_demo.ipynb

```diff
@@ -0,0 +1,609 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "total 1012\n",
+      "336 -rw-rw-r-- 1 ubuntu ubuntu 340551 Oct  5 20:46 model.bin\n",
+      "  4 drwxrw-rwx 2 ubuntu ubuntu   4096 Oct  5 20:46 movie\n",
+      "668 -rw-rw-r-- 1 ubuntu ubuntu 681259 Oct  5 20:46 optimizers.bin\n",
+      "  4 drwxrw-rwx 2 ubuntu ubuntu   4096 Oct  5 20:46 user\n"
+     ]
+    }
+   ],
+   "source": [
+    "# check the saved artifacts\n",
+    "!ls -ls {best_model_path}"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3. Inference with the trained model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/opt/conda/envs/pytorch/lib/python3.10/site-packages/torch/_utils.py:776: UserWarning: TypedStorage is deprecated. It will be removed in the future and UntypedStorage will be the only storage class. This should only matter to you if you are using storages directly.  To access UntypedStorage directly, use tensor.untyped_storage() instead of tensor.storage()\n",
+      "  return self.fget.__get__(instance, owner)()\n",
+      "INFO:root:successfully load the model from nc_model/epoch-2\n",
+      "INFO:root:Time on load model: 0.010 seconds\n"
+     ]
+    }
+   ],
+   "source": [
+    "# we can restore the model from the checkpoint:\n",
+    "model.restore_model(best_model_path)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "INFO:root:Start to load partition from /mnt/efs/gsf-data/movielen_100k_train_val_1p_4t/part0/graph.dgl which is 5839353 bytes. It may take non-trivial time for large partition.\n",
+      "INFO:root:Finished loading partition.\n",
+      "INFO:root:Finished loading node data.\n",
+      "INFO:root:Finished loading edge data.\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Create a dataset for inference, we use the same MovieLens graph\n",
+    "infer_data = GSgnnNodeInferData(\n",
+    "    config.graph_name, \n",
+    "    config.part_config,\n",
+    "    eval_ntypes=config.target_ntype,\n",
+    "    node_feat_field=config.node_feat_name,\n",
+    "    label_field=config.label_field)\n",
+    "\n",
+    "# Set up dataloader for the inference dataset\n",
+    "infer_dataloader = GSgnnNodeDataLoader(\n",
+    "    infer_data, infer_data.test_idxs, fanout=config.fanout, \n",
+    "    batch_size=100, device=device,\n",
+    "    train_task=False)\n",
+    "\n",
+    "\n",
+    "# Create an Inferrer object\n",
+    "infer = GSgnnNodePredictionInferrer(model)\n",
+    "infer.setup_device(device=device)\n",
+    "infer.setup_evaluator(evaluator)\n"
```

Comment:
Also set task_tracker to show testing results during inference.

Reply:
https://github.com/awslabs/graphstorm/pull/530#discussion_r1350650746